.idea/copyright/GPL.xml

```diff
@@ -0,0 +1,6 @@
+<component name="CopyrightManager">
+  <copyright>
+    <option name="notice" value="Copyright (C) &amp;#36;today.year &lt;Your Name&gt;&#10; &#10;This program is free software: you can redistribute it and/or modify&#10;it under the terms of the GNU General Public License as published by&#10;the Free Software Foundation, either version 3 of the License, or&#10;(at your option) any later version.&#10;&#10;This program is distributed in the hope that it will be useful,&#10;but WITHOUT ANY WARRANTY; without even the implied warranty of&#10;MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the&#10;GNU General Public License for more details.&#10;&#10;You should have received a copy of the GNU General Public License&#10;along with this program.  If not, see &lt;https://www.gnu.org/licenses/&gt;." />
+    <option name="myName" value="GPL" />
```

Comment:
Not LPGL?

Reply:
You are right.. LGPL is just a special version of the GPL. The copyright text comes from the GPL license but is not contained in the LGPL. The latter one references the GPL in some points. I will change this. Maybe we should discuss the license again and use GPL only to make things easier.

Reply:
I am ok with GPL.

Reply:
Would be fine with me

Reply:
The main difference between GPL and LGPL is for me that with LGPL your software must not be open, if you just link to a library that has the LGPL license. Hence, for this app it make probably more sense to switch to GPL because be don't provide a library. I will change the license as part of this PR.